src/altinn-app-frontend/src/features/form/components/MessageBanner.test.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+
+import MessageBanner from './MessageBanner';
+
+describe('form/components/MessageBanner.tsx', () => {
```

Comment:
```suggestion
describe('features / form / components / MessageBanner.tsx', () => {
```

To be consistent with test naming. Don't know how establised this is, but atleast it's consistent with newer tests + Legend.test.tsx